hooks/pre_commit/13-tuning.rb

```diff
@@ -0,0 +1,76 @@
+# A mapping of tuning profile to the required CPU cores and memory in GB
```

Comment:
Should the logic of this whole file be also wrapped in the `has_custom_fact?('tuning')` check?

Reply:
It's already checking `if app_option?(:tuning)` which I think is cleaner. If we decide to expose the option on a different condition, it'll still work. I'm not sure about the constants.

Reply:
I'll push an update wrapping the constants as well